participants/l10n_update_ts.py

```diff
@@ -226,22 +236,28 @@ def init_gitdir(self, reponame):
                     'has_downloads': False,
                     'auto_init': True
                 }
-                ghresp = requests.post("%s/user/repos" % self.gitconf["apiurl"],
-                                       auth=gitserv_auth,
-                                       headers={
-                                           'content-type': 'application/json'
-                                       },
-                                       verify=False,
-                                       data=json.dumps(payload))
+                ghresp = requests.post(
+                    "%s/user/repos" % self.gitconf["apiurl"],
+                    auth=gitserv_auth,
+                    headers={'content-type': 'application/json'},
+                    verify=False,
+                    data=json.dumps(payload)
+                )
                 ghresp.raise_for_status()
 
-            check_call(["git", "clone",
-                        self.gitconf["repourl"] % {
-                            "username": self.gitconf["username"],
-                            "reponame": reponame
-                        }],
+            repo_url = self.gitconf["repourl"] % {
+                "username": self.gitconf["username"],
+                "reponame": reponame
+            }
+            check_call(["git", "clone", repo_url, gitdir],
                        cwd=self.gitconf["basedir"])
```

Comment:
So this clones a remote git repo and checks out the branch set by the remote (could be master or anything else)